springwolf-core/src/main/java/io/github/stavshamir/springwolf/configuration/AsyncApiDocket.java

```diff
@@ -18,9 +18,13 @@ public class AsyncApiDocket {
     /**
      * The base package to scan for listeners.
      */
-    @NonNull
     private String basePackage;
```

Comment:
I think we should make the comments more clear, as `basePackage` and `configurationBasePackage` are very similar and may be confusing. 

I think this should change to: 
```java
/**
 * The base package to scan for listeners which are declared inside a class annotated with @Component or @Service.
*/
```